repo/packages/H/hdfs/0/config.json

```diff
@@ -26,7 +42,10 @@
      },
      "required": [
       "scheduler-uris",
-      "framework-name"
+      "framework-name",
```

Comment:
We need to use dashes, not underscores here. That is the consistent format used in the universe packaging.